apps/condo/domains/banking/schema/BankSyncTask.js

```diff
@@ -135,10 +145,44 @@ const BankSyncTask = new GQLListSchema('BankSyncTask', {
         },
     },
     hooks: {
+        beforeChange: async ({ resolvedData, context, operation }) => {
+            if (operation === 'create' && get(resolvedData, 'options.type') === 'SBBOL') {
+                if (!resolvedData.account) {
+                    throw new GQLError(ACCOUNT_IS_REQUIRED, context)
+                }
```

Comment:
Could you move this validation into `validateInput` hook of `account` field.

Conceptually, `beforeChange`, `afterChange` hooks are used for side effects.
In your case this is not a side effect, but a validation.

The code below, that implements side effect of updating `BankAccount` will be naturally executed after all validation has been passed.

You can take a short look at execution order in Keystone documentation
https://v5.keystonejs.com/guides/hooks#execution-order